open-hackathon-server/src/hackathon/hmongo/models.py

```diff
@@ -128,6 +150,11 @@ class UserToken(DynamicDocument):
     def __init__(self, **kwargs):
         super(UserToken, self).__init__(**kwargs)
 
+    # TODO: should inherit this from HDocumentBase? then should we delete the default
+    # fields in the HDocumentBase?
+    def dic(self):
```

Comment:
The `UserToken` doesn't inherit from `HDocumentBase`, so I have to re-define one